candle-transformers/src/models/t5.rs

```diff
@@ -633,9 +644,22 @@ impl T5ForConditionalGeneration {
         decoder_cfg.num_layers = cfg.num_decoder_layers.unwrap_or(cfg.num_layers);
         let decoder = T5Stack::load(true, vb.pp("decoder"), &shared, &decoder_cfg)?;
 
+        let tie_word_embeddings = cfg.tie_word_embeddings;
+        let lm_head = match tie_word_embeddings {
```

Comment:
Use a `if` when matching on a `bool` if possible (it's what we do everywhere else in this codebase).

Reply:
Done. PTAL.